README.md

```diff
@@ -1 +1,56 @@
-# php-scrypt
\ No newline at end of file
+#  scrypt-php
+
+A PHP Extension written in Rust to create scrypt password hashes
+
+## Usage
+Using the scrypt extension is easy, just add the extension to your PHP installation and use the scrypt functions like so:
```

Comment:
We should show how to install the extension. Can we have a new 'installation' section?  Simply showing how to put SO file into PHP server.

With that, maybe we should have a Dockerfile, and docker-compose? So people can easily try it out.

We should probably also have releases, so people can download so file from there, instead of building. Is there some SO file package manager where we should push it? 🤔

Reply:
I can add some github actions to automatically build for platforms if you like. As for package managers, I have no idea. Maybe PECL?

Reply:
I've added an installation section to the readme